### PR TITLE
PPTP-1851 Returns sends signed in user for uplift if they do not have strong credentials

### PR DESCRIPTION
**Pull request by tonymccrae-ee**
### Description of Work carried through

Returns front end requires strong credentials for consistency with Registration.

1 failing acceptance test looks unrelated.

#### Check list 
 - [x] `./precheck` was executed (Integration/Component/Unit tests)
 - [x] `sbt scalafmt test:scalafmt` was executed 
 - [ ] Required Environment Config has been amended/added
 - [ ] Links to dependencies have been included (BE/FE work)
 - [ ] User Acceptance Tests (UAT) were run locally and have passed.
 - [ ] No Accessibility errors/warnings reported by Wave

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -135,6 +135,7 @@ urls {
   exitSurvey = "http://localhost:9514/feedback/plastic-packaging-tax-registration"
   login = "http://localhost:9949/auth-login-stub/gg-sign-in"
   loginContinue = "http://localhost:8505/plastic-packaging-tax/account"
+  mfaUplift = "http://localhost:9553/bas-gateway/uplift-mfa"
```
Comment (tonymccrae-ee) [inline conf/application.conf:138]: Spot the thing which will need to be pushed out to all the env config...

File: test/uk/gov/hmrc/plasticpackagingtax/returns/controllers/actions/AuthActionSpec.scala [hunk 2]
```diff
@@ -156,7 +162,7 @@ class AuthActionSpec extends ControllerSpec with MetricsMocks {
       authorizedUser(user)
 
       await(createAuthAction().invokeBlock(authRequest(Headers(), user), okResponseGenerator))
-      metricsMock.defaultRegistry.timer("ppt.returns.upstream.auth.timer").getCount should be > 1L
+      metricsMock.defaultRegistry.timer("ppt.returns.upstream.auth.timer").getCount should be > 0L
```
Comment (tonymccrae-ee) [inline test/uk/gov/hmrc/plasticpackagingtax/returns/controllers/actions/AuthActionSpec.scala:165]: This looks like a was passing by luck / previous test run order assert; taken to mean, the time was called at least once.

File: conf/application.conf (hunk 1 above)
Comment (paulblakehmrc) [reply conf/application.conf:138]: Could just add it to base config repo instead, just like the reg FE

**Review by paulblakehmrc — APPROVED** [review]
LGTM

**Pull request merged**